assignments/this.js

```diff
@@ -1,26 +1,57 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit binding the this keyword is referring to whatever is to the left of the dot. 
+* 2. Explicit binding uses call, apply and bind to say what a function is going to be. Call makes you indicate each argument individually, apply allows you to use an array and bind gives you a new array you can call later.
+* 3. New binding when used, a new object is created inside of the function. 
+* 4. Window binding is the default use of the this keyword. If nothing is specified, window binding seaches the entire environment. This keyword is used outside of a function.
 *
 * write out a code example of each explanation above
 */
 
-// Principle 1
+// Principle 1 - Window (Global)
 
-// code example for Window Binding
+function(window) {
```

Comment:
You're very close here! The way you write functions in ES5 syntax is: `function funcName(){...}`
What do you think you are missing here?
You are also passing in an argument into this function - what is the goal in it??

Reply:
I think it's missing a funcName. I misplaced the funcName, which is supposed to be 'window' and no argument passed.